CBLClient/Apps/CBLTestServer-Dotnet/TestServer/FileLoggingMehtod.cs

```diff
@@ -156,6 +157,42 @@ public class FileLoggingMehtod
 
         }
 
+        public static void GetLogsInZip([NotNull] NameValueCollection args,
+                                [NotNull] IReadOnlyDictionary<string, object> postBody,
+                                [NotNull] HttpListenerResponse response)
+        {
+            var path = Database.Log.File.Config?.Directory;
+            if (path == null) {
+                response.WriteEmptyBody();
+                return;
+            }
+
+            string[] filePaths = Directory.GetFiles(@path);
+            var memoryStream = new MemoryStream();
+            using (var archive = new ZipArchive(memoryStream, ZipArchiveMode.Create, true)) {
+                foreach (var filePath in filePaths) {
+                    var entry = archive.CreateEntry(Path.GetFileName($"{filePath}.zip"));
+                    using (var file = File.Open(filePath, FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+                    using (var entryStream = entry.Open()) {
+                        file.CopyTo(entryStream);
+                    }
+                }
+            }
+
+            try {
+                response.ContentLength64 = memoryStream.Length;
+                Stream output = response.OutputStream;
+                memoryStream.WriteTo(output);
```

Comment:
does this means that method is sending binary data on the channel and user has to write it back into a file and save  it as zip at client side?

Reply:
Yes that is what it means